QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -19,9 +19,15 @@
 import ccre.log.Logger;
 
 public class Clamp {
-    public static FloatStatus heightOrSpeed = new FloatStatus();
+    public static FloatStatus deltaHeightOrSpeed = new FloatStatus();
```

Comment:
delta height? When it's in height mode, this controls the absolute height, not delta